v2/Dockerfile

```diff
@@ -1,10 +1,29 @@
-# Note: This Dockerfile assumes that the binary has been built using the top-level Taskfile.yml
```

Comment:
Note that this Dockerfile still assumes that the `bundle-crds` taskfile target has been run, for line 27 to do the right thing. Might be worth keeping a comment either here at the top or on line 27 for the CRD copy?